src/google-api-typings-generator.ts

```diff
@@ -161,7 +161,7 @@ class TypescriptTextWriter implements ITypescriptTextWriter {
 
     public comment(text: string = "") {
         var lines = text.split(/\r\n|\r|\n|\u000a\u000d|\u000a|\u000d|\u240a/g);
-        _.forEach(lines, line => this.writer.writeLine(`// ${line}`));
+        _.forEach(lines, line => this.writer.writeLine(`/** ${line} */`));
```

Comment:
Since this is emitted for each line, don't you want something like the following?

```ts
this.writer.writeLine(`/**`);
_.forEach(lines, line => this.writer.writeLine(` * ${line}`);
this.writer.writeLine(`*/`);
```